ocp_resources/forkliftcontroller.py

```diff
@@ -0,0 +1,35 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class ForkliftController(NamespacedResource):
+    """
+    Migration Toolkit For Virtualization (MTV) ForkliftController object
+    """
+
+    api_version = NamespacedResource.ApiVersion.V1BETA1
```

Comment:
Do we have api_group for this resource?
Better to use it if we have, since then we can grub the api_version automatically

Reply:
Actually we do have. I will use it, thanks.

Reply:
Done